package/MDAnalysis/analysis/hydrogenbonds/hbond_analysis.py

```diff
@@ -280,9 +280,9 @@ def __init__(self, universe,
             universe topology does not contain bonding information
         d_a_cutoff : float (optional)
             Distance cutoff for hydrogen bonds. This cutoff refers to the D-A distance. [3.0]
-        d_h_a_angle_cutoff: float (optional)
+        d_h_a_angle_cutoff : float (optional)
```

Comment:
forgot to remove defaults -- the numbers in square brackets!

Reply:
Done